lib/droplet_kit/mappings/kubernetes_node_pool_mapping.rb

```diff
@@ -0,0 +1,19 @@
+module DropletKit
+  class KubernetesNodePoolMapping
+    include Kartograph::DSL
+
+    kartograph do
+      mapping KubernetesNodePool
+      root_key plural: 'node_pools', scopes: [:read]
```

Comment:
we could reuse this mapping in the clusters all endpoint which will be nice 👍

Reply:
👍

Reply:
worth noting, in order to have a single node poool mapping you need to add `singular: 'node_pool'` here so that it knows what to key off of.